modules/twinklexfd.js

```diff
@@ -716,8 +723,8 @@ Twinkle.xfd.callbacks = {
 			var text = pageobj.getPageText();
 			var params = pageobj.getCallbackParameters();
 
-			pageobj.setPageText((params.noinclude ? "<noinclude>" : "") + "{{subst:template for discussion|help=off|" +
-				(params.tfdtype !== "standard" ? "type=" + params.tfdtype + "|" : "") + mw.config.get('wgTitle') + (params.noinclude ? "}}</noinclude>" : "}}") + text);
```

Comment:
Is `mw.config.get('wgTitle')` here ever necessary?  I don't see that it is.

Reply:
It could be optionally passed to `|page=`. I think it may have been `|1=` at some point.

Reply:
From looking at [[Template:Template for discussion/dated]], it seemed like it was used for optionally changing the section linked in the template.  That really only makes sense when bulk nominating, which twinkle doesn't handle yet, so I think it's fine to remove.

Reply:
> I think it's fine to remove.

Agreed

Reply:
Worth noting too that `taggingTemplateForMerge` likewise doesn't have it.